data/coalition/heliarch intro.txt

```diff
@@ -1699,7 +1699,7 @@ mission "Heliarch Drills 2-A"
 				`	"Okay, I'll head out to the planets then."`
 					goto accept
 				`	"Can you tell me where my pursuers will be?"`
-			`	"Know of their pursuers, the pursued do not," she says, letting out a laugh as if it was a joke. "Well, usually... From many systems across the Coalition, the Hunters are. Ready to follow your ship when you launch, they will be."`
+			`	"Know of their pursuers, the pursued do not," she says, letting out a laugh as if it was a joke. "Well, usually... from many systems across the Coalition, the Hunters are. Ready to follow your ship when you launch, they will be."`
```

Comment:
```suggestion
			`	"Know of their pursuers, the pursued do not," she says, letting out a laugh as if it was a joke. "Well, usually... From many systems across the Coalition, the Hunters are. Ready to follow your ship when you launch, they will be."`
```

I *think* this one is correct as is because Lotuora isn't continuing on as if explaining after the ...
As in, she's not saying "Well, usually from many systems across the Coalition, the Hunters are", she's trailing off, with the "Well, usually" complementing the previous "Know of their pursuers, the pursued do not" sentence. "From many systems across the Coalition, the Hunters are" is a whole new sentence, independent from the previous one and the "Well, usually", so I'd guess this one would remain with the capitalized F.

If that's not how it works though even if it is a new sentence, then ignore this.